BioCro/DARPA/partitioned_biomass_data.Rmd

```diff
@@ -33,29 +33,28 @@ opt_weather_plot <- opt_weather %>%
   mutate(date = seq(as.POSIXct("2019-01-01 00:00:00"), 
                     as.POSIXct("2019-12-31 23:00:00"), 
                     by = "hour")) %>% 
-  dplyr::select(solarR:date) %>% 
+  dplyr::select(solarR:date) %>%
   tidyr::pivot_longer(cols = solarR:precip)
 
 ggplot(opt_weather_plot, aes(date, value)) +
-  geom_line(size = 0.1) + 
+  geom_line(size = 0.1) +
   facet_wrap(~name, ncol = 1, scales = 'free_y')
 
 ggplot(opt_weather_plot %>% filter(month(date) ==1 & day(date) < 3), aes(date, value)) +
-    geom_line() + 
+    geom_line() +
     facet_wrap(~name, ncol = 1, scales = 'free_y')
 ```
 
 ### Biomass data
 
-[These data](https://docs.google.com/spreadsheets/d/1Nc2g-gPEb-rUW9R4QLDqAZvRRYk1whWAJlb8kLEQZ5c/edit#gid=0) are in the project's Google Drive. They have been downloaded and saved as `biomass_setaria_me034_gehan.csv`. 
-
-Plants all started to grow at same time (Jan 3, 2019). Plants harvested on six different dates, from Jan 15 - Feb 19. The plants were grown at three different temperatures, and there were three genotypes. Initially only using data for the middle temperature (31*C) and taking the median of the three values. Biomass measurements are from destructively harvesting plants, so plants from different harvest dates are different plants.  
+These data are in [the project's private data repo](https://github.com/az-digitalag/model-vignettes-data). Plants all started to grow at same time (Jan 3, 2019). Plants harvested on six different dates, from Jan 15 - Feb 19. The plants were grown at three different temperatures, and there were three genotypes. Initially only using data for the middle temperature (31*C) and taking the median of the three values. Biomass measurements are from destructively harvesting plants, so plants from different harvest dates are different plants.  
```

Comment:
Maybe provide instructions for checking out these data? like 

```sh
git clone https://github.com/az-digitalag/model-vignettes-data ../model-vignettes-data
```

and then proceed assuming that the data are available under ../model-vignettes-data (or to copy files to where this Rmd expects them?)